.env

```diff
@@ -0,0 +1,5 @@
+DB_HOST = 'localhost'
+DB_PW = ''
+DB_USER = 'root'
+DB_NAME = 'storiesdb'
+SECRET_KEY = 'xT1tdO3CfMH01pjxC+guN1LWSt2nKvr5td6KUpw7Czg='
```

Comment:
best practice would dictate not committing secrets like this to the repo

github.com even has a way to securely store secret values like this, I believe.  or you could always just state in the README that such a value needs to be added/created after checkout/clone